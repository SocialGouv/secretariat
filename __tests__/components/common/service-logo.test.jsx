import { render } from "@testing-library/react"

import ServiceLogo from "../../../src/components/common/service-logo"

import { it, expect } from "vitest"

it("renders small service logo", () => {
  const { container } = render(<ServiceLogo size="sm" name="matomo" />)
  expect(container).toMatchSnapshot()
})

it("renders medium service logo", () => {
  const { container } = render(<ServiceLogo name="matomo" />)
  expect(container).toMatchSnapshot()
})

it("renders large service logo", () => {
  const { container } = render(<ServiceLogo size="lg" name="matomo" />)
  expect(container).toMatchSnapshot()
})
