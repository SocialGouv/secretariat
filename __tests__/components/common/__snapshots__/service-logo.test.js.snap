// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders large service logo 1`] = `
<div>
  <img
    alt="matomo logo"
    class=""
    data-nimg="1"
    decoding="async"
    height="24"
    loading="lazy"
    src="/_next/image?url=%2Fimages%2Fdark%2Fmatomo.svg%23icon-logo&w=96&q=75"
    srcset="/_next/image?url=%2Fimages%2Fdark%2Fmatomo.svg%23icon-logo&w=48&q=75 1x, /_next/image?url=%2Fimages%2Fdark%2Fmatomo.svg%23icon-logo&w=96&q=75 2x"
    style="color: transparent;"
    title="matomo"
    width="34.896"
  />
</div>
`;

exports[`renders medium service logo 1`] = `
<div>
  <img
    alt="matomo logo"
    class=""
    data-nimg="1"
    decoding="async"
    height="22"
    loading="lazy"
    src="/_next/image?url=%2Fimages%2Fdark%2Fmatomo.svg%23icon-logo&w=64&q=75"
    srcset="/_next/image?url=%2Fimages%2Fdark%2Fmatomo.svg%23icon-logo&w=32&q=75 1x, /_next/image?url=%2Fimages%2Fdark%2Fmatomo.svg%23icon-logo&w=64&q=75 2x"
    style="color: transparent;"
    title="matomo"
    width="31.988"
  />
</div>
`;

exports[`renders small service logo 1`] = `
<div>
  <img
    alt="matomo logo"
    class=""
    data-nimg="1"
    decoding="async"
    height="18"
    loading="lazy"
    src="/_next/image?url=%2Fimages%2Fdark%2Fmatomo.svg%23icon-logo&w=64&q=75"
    srcset="/_next/image?url=%2Fimages%2Fdark%2Fmatomo.svg%23icon-logo&w=32&q=75 1x, /_next/image?url=%2Fimages%2Fdark%2Fmatomo.svg%23icon-logo&w=64&q=75 2x"
    style="color: transparent;"
    title="matomo"
    width="26.172"
  />
</div>
`;
